lib/SIL/SILBasicBlock.cpp

```diff
@@ -355,3 +355,7 @@ bool SILBasicBlock::isTrampoline() const {
     return false;
   return begin() == Branch->getIterator();
 }
+
+bool SILBasicBlock::isLegalToHoistInto() const {
+  return true;
```

Comment:
@gottesmm, is this the right implementation for now? If so I'll extract it out into a separate PR so we can land it and be done.